cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/DistributedStreamService.java

```diff
@@ -534,9 +551,8 @@ protected StreamSizeAggregator(Id.Stream streamId, long baseCount, int streamThr
       this.cancellable = cancellable;
       this.isInit = true;
       this.streamFeed = new Id.NotificationFeed.Builder()
-        .setNamespaceId(streamId.getNamespaceId())
-        .setCategory(Constants.Notification.Stream.STREAM_FEED_CATEGORY)
-        .setName(streamId.getName())
+        .setNamespaceId(streamId.getNamespaceId()).setCategory(Constants.Notification.Stream.STREAM_FEED_CATEGORY)
+        .setName(String.format("%sSize", streamId.getName()))
```

Comment:
isn't this easily resolvable by using a different Category for sizes, and leaving the name of the feed simply the name of the stream? (possibly JIRA for later, since its not directly related to the focus of this PR).